sdk/servicebus/Azure.Messaging.ServiceBus/tests/SessionReceiverLiveTests.cs

```diff
@@ -318,6 +318,174 @@ await using (var scope = await ServiceBusScope.CreateWithQueue(enablePartitionin
             }
         }
 
+        [Test]
+        public async Task CompleteMessages()
+        {
+            await using (var scope = await ServiceBusScope.CreateWithQueue(enablePartitioning: false, enableSession: true))
+            {
+                await using var sender = new ServiceBusSenderClient(TestEnvironment.ServiceBusConnectionString, scope.QueueName);
+                var messageCount = 10;
+                var sessionId = "sessionId1";
```

Comment:
Can we use test cases to get coverage for both session specified and not specified?

Reply:
I've covered both the scenarios.  I've added session specified test cases in `SessionReceiverLiveTests` class and non session specified test cases in `ReceiverLiveTests` class.

Reply:
I'm talking about sessionful entities where you don't specify a particular sessionId. Do we have a test for that?

Reply:
Ohh my bad, I thought you're taking about non session entities. 

I'll add the test cases for session entities where we don't specify session Id.

Reply:
No worries, my initial comment wasn't clear :)